src/main/java/name/valery1707/kaitai/KaitaiGenerator.java

```diff
@@ -49,13 +49,13 @@ public String getPackageName() {
 		return unmodifiableSet(sources);
 	}
 
-	public KaitaiGenerator withSource(Path source) throws MojoExecutionException {
+	public KaitaiGenerator withSource(Path source) throws KaitaiException {
```

Comment:
\[Checkstyle\] WARNING: 'method def modifier' has incorrect indentation level 8, expected level should be 2\.